packages/teams-js/src/private/files.ts

```diff
@@ -763,7 +834,7 @@ export namespace files {
       throw new Error('[registerCloudStorageProviderListChangeHandler] Handler cannot be null');
     }
 
-    registerHandler('files.cloudStorageProviderList', handler);
+    registerHandler('files.cloudStorageProviderListChange', handler);
```

Comment:
this is fine, but keep in mind you will also need to update the other sdk

Reply:
Sure.